PIM/API/04_RetrieveEntityData.md

```diff
@@ -0,0 +1,213 @@
+# Retrieve entity data
+
+To make an API call to the *Actindo Core OpenAPI*, you usually need to provide one or several entity IDs in your request body, such as product ID or attribute set ID, for the system to recognize the entity you are addressing. The ID, that is, the entity identification number, is automatically assigned by the system when creating an entity.
+
+For example, when you want to create a product, you need to send an *attributeSetId*, that is, the identification number of the attribute set of the product you want to create. You can find out an entity ID via user interface or via API. Besides, you can also retrieve other specific entity data via API if desired.
+
+In the following, a few examples to retrieve entity data via user interface and via API are provided. 
+
+
+## Discover an entity ID via user interface
+
+You can find out any entity ID via user interface as follows:
+
+1. Select the module where the entity originates from, for example, *PIM > Products > LIST* to find out a product ID.
+
+2. Locate the entity whose ID you need.
+
+3. Check the ID in the *ID* column. 
+
+    > [Info] If the *ID* column is hidden, see [Add or remove columns](../../Core1Platform//UsingCore1/05_WorkWithLists.md#add-or-remove-columns) to learn how to display it. 
+
+    ![Entity ID in user interface](../../Assets/Screenshots/PIM/API/EntityID_UI.png "[Entity ID in user interface]")
+
+4. Write down the entity ID. You will need to provide it when sending your request.
+
+
+## Discover an entity ID via API
+
+Generally, you can get all entity data you need via API. The entity data is provided in the response to your request. Depending on the entity data you need, the endpoint you have to address may vary. 
+
+If desired, you can also set filters to narrow down your request response. See [List products](./05_ManageProducts.md#list-products) for the filter definitions and request samples.
+
+In the following, a few useful use cases, with their corresponding request samples, are provided.  Refer to the API documentation in your instance under *Dev Tools > API > Module name* for a complete list of endpoints.
+
+
+### List the PIM attribute sets
+
+Get a list of all *PIM* attribute sets in your current instance, including the attribute set IDs.
+
+#### Definitions
+
+| Attribute key  | Data type | Description | 
+| ---------------|-----------|-------------| 
+| **dataType**   | string    | Entity data type |   
+
+**Endpoint**: /Actindo.Modules.Actindo.PIM.AttributeController.getListOfAttributeSets
+
+#### Request sample
+
+        {
+        "dataType": "Actindo\\Modules\\Actindo\\DataHub\\DataTypes\\IntegerValue"
+        }
+
+> [Info] Since the attribute set ID is a whole number, you have to include the data type *Actindo\\Modules\\Actindo\\DataHub\\DataTypes\\IntegerValue* in your request.  
```

Comment:
```suggestion
> [Info] Since the attribute set ID is a whole number, you have to include the *Actindo\\Modules\\Actindo\\DataHub\\DataTypes\\IntegerValue* data type in your request.  
```